### DISCUSSION TITLE
International sites

### DISCUSSION THREAD

**Discussion by LParkhill** [NREL/REopt-API-Analysis#24] [2 - Site & Utility Questions] [answered]
Can I use the REopt web tool for locations outside of the United States?

**Comment by LParkhill** [comment, marked as answer]
The REopt® web tool was developed for commercial sites in the United States, but it can be used for International sites with some modifications:

**Site Location & Utility Rate**
Selecting a site location outside the US will prompt a message that no electricity rates can be found for the location. This is because the utility rate database used by the REopt web tool does not include international locations. However, custom utility rates can be entered as simple annual or monthly rates. Detailed rates, with variable prices dependent on times and months, can also be entered if the user is registered and logged in to a user account. Details of rate structures for some international locations can be found at the [International Utility Rate Database](https://openei.org/apps/IURDB/).
 
**Currency**
Currency values are all in US dollars. Conversions from the local currency to USD can be made for inputs of utility rates, system costs, and incentive values. Conversion of the final results of the evaluation will then be necessary, from USD back to the local currency. One popular tool for currency conversion approximation is the [Currency.Wiki](https://www.currency.wiki/).
 
**Load Profile**
The Load Profile option for simulated load data is based on US building and climate area data. If this simulated load option is used, the simulated load profile should be checked for reasonableness for the climate of the selected location.
 
**Financial Info**
Financial, tax and incentive input defaults in all sections need to be carefully considered and altered to match local tax and interest rates and available financial incentives. Default costs for technology systems are also based on typical costs in the US. Resources for researching international renewable energy costs can be found at the [International Renewable Energy Agency](https://www.irena.org/costs).
 
**Solar Production Data**
Solar production data is taken from the PVWatts® dataset, which includes many international locations. The REopt web tool will use the closest available location that is found to have resource data, so the user should independently confirm that PVWatts includes data for a location that is acceptably close to their site location. The available resource data locations can be found using [NREL's PV Watts](https://pvwatts.nrel.gov/pvwatts.php). Custom PV production factors can be used in place of the PVWatts data by uploading a user-defined annual PV generation profile.
 
**Wind Resource Data**
Wind systems can’t currently be modeled from the web tool user interface for international locations due to lack of international wind resource data. However, if the user has hourly wind resource data for their site, they can use this data in the application programming interface (API), instead of the web tool interface, to complete an optimization.

**Comment by xinyan915** [comment]
Where can I upload the "solar production data"?

**Reply by LParkhill** [reply]
The solar production data, or "PV generation profile" can be uploaded as a .csv file in the "advanced inputs" part of the PV section. The file needs to be normalized to units of kWh/kW, representing the AC energy (kWh) output of a 1 kW-DC system in each time step. The file must be one year (January through December) of hourly, 30-minute, or 15-minute PV generation data. For example, if you have an 8760 hour file of the kWh-AC production for a 200 kW-DC system, you would divide each hour's kWh production by 200 kW to get 8760 hours of kWh/kW production of a 1 kW system.
